test/test_torch.py

```diff
@@ -3809,11 +3809,11 @@ def tensorfn(myfn, t1, t2):
                     if fn == "lerp":
                         return myfn(t1, 0.5)
                     elif fn == "masked_select":
-                        return myfn(t1 < 0)
+                        return myfn((t1 < 0).to(torch.bool))
```

Comment:
As above, I'd avoid these changes because the comparison op change will handle this for free.